docs/marketplace/dev/front-end/angular-services.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Angular Services
+description: This articles provides details about what are Angular Services, how to use them and how to create them.
+template: concept-topic-template
+---
+
+This articles provides details about what are Angular Services, how to use them and how to create them.
+
+## Introduction
+
+Angular services are singleton objects that get instantiated only once during the lifetime of an application. They contain methods that maintain data throughout the life of an application, i.e. data does not get refreshed and is available all the time. The main objective of a service is to organize and share business logic, models, or data and functions with different components of an Angular application.
```

Comment:
Services are not always singletons - it depends how they are provided in the app and can be instantiated multiple times.